src/slider/slider.scss

```diff
@@ -3,7 +3,7 @@
 @import '../style/index';
 @import '../icon/index';
 
-$thumb-height: $iui-m; // 16px
+$thumb-height: $iui-m + $iui-xxs; // 18px to account for border
```

Comment:
hm.. Maybe we should keep 16px? Not sure..

Reply:
Oh wait.. I am lost.. even though you were adjusting sizes, images changed. So maybe didnt need to do that?
i.e. slider image now looks like thumb is not centered (or it was not centered) ?

Reply:
16px wasn't centered vertically. The images changed because the slider is not in the same place horizontally. If it doesn't look vertically centered, it was that way before.

Reply:
But it is not centered horizontally because it can be seen clearly that 50% mark is not in the middle of the thumb. You might need to tweak this one 
![image](https://user-images.githubusercontent.com/36186912/171370276-d8428f37-ca3d-4032-a4cd-d891c2d60c64.png)